README.md

```diff
@@ -141,7 +144,7 @@ A `RequirementType` of `All` changes the traversal. First, if there are no filte
 
 In the above example, `FeatureW` specifies a `RequirementType` of `All`, meaning all of it's filters must evaluate to true for the feature to be enabled. In this case, the feature will be enabled for 50% of users during the specified time window.
     
-### Service Registration
+## Service Registration

```

Comment:
I think the root title should be **Consumption** and then it goes over `featureManager.IsEnabledAsync` right away.

Then it Service Registration, and DI can be following subsections.

Feature check section would no longer be needed.

And I'd suggest moving the scoped services section last in the **Consumption** parent section.

So

Consumption
  * Service Registration
  * Dependency Injection
  * Scoped Feature Management Services